src/engineer/base.py

```diff
@@ -54,7 +54,8 @@ def _process_static(self):
         # lat and lon
 
         output_file = self.static_output_folder / 'data.nc'
-        assert not output_file.exists(), 'A static data file already exists!'
+        if not output_file.exists():
+            warnings.warn('A static data file already exists!')
```

Comment:
This should be `if output_file.exists()` (it's currently `if not`).

Also, after the warning perhaps there could be a `return None` to make sure the static stuff isn't run (since it won't work)

Reply:
Good point thankyou